source/extensions/resource_monitors/cpu_utilization/linux_cpu_stats_reader.h

```diff
@@ -29,13 +37,19 @@ class LinuxContainerCpuStatsReader : public CpuStatsReader {
   LinuxContainerCpuStatsReader(
       TimeSource& time_source,
       const std::string& linux_cgroup_cpu_allocated_file = LINUX_CGROUP_CPU_ALLOCATED_FILE,
-      const std::string& linux_cgroup_cpu_times_file = LINUX_CGROUP_CPU_TIMES_FILE);
+      const std::string& linux_cgroup_cpu_times_file = LINUX_CGROUP_CPU_TIMES_FILE,
```

Comment:
I don't think there's a scenario where we'd use both cgroupv1 and cgroupv2 at the same time -- I'm not sure if it's even possible.

Can we do something similar to https://github.com/envoyproxy/envoy/blob/4c5d150710ba38add0acfcc2dff6a1edb2f363f1/source/extensions/resource_monitors/cgroup_memory/cgroup_memory_stats_reader.h#L79-L98 and have a seperate reader for the different implementations?